### ISSUE TITLE
Error using .ttc font Kohinoor Devanagari on OS X 10.10

### ISSUE THREAD

**Issue by deepakjois** [simoncozens/sile#75]
Kohinoor Devanagari is the default Devanagari font included on OS X 10.10. This code however fails to build with error: 
```
libtexpdf:fatal: sfnt: table not found..
```

```
\begin[papersize=a5]{document}
\script[src=packages/bidi]
\font[family=Kohinoor Devanagari,size=10pt,language=hi]
यह दाग़ दाग़ उजाला
\end{document}
```

Similar errors are happening in XeTeX on Tex Live on OS X as well:
http://tug.org/pipermail/xetex/2015-April/025874.html

Because `libtexpdf` has been extracted from `xdvipdfmx`, the fixes should be ported over from there as well.

**Comment by simoncozens** [comment]
Thanks. I'm aware of the thread on the xetex mailing list, and I'll pull the change from the dvipdfmx repository soon.

**Comment by alerque** [comment]
I've been going through the test suit making sure stuff is actually sane and hoping to setup the build tests to actually run the regression options. I've been unable to duplicate this bug through. I've tried:

* Working on an OSX 10.10 system
* Checking out SILE at 807d7ff (right before the fix)
* Making sure the submodule was indeed at 591d662 (which is actually several before the fix which purports to be in 933e0a6)
* Recompiling with `./configure ; make`
* Grabbing a copy of the test file out of a later revision
* Running SILE locally with `DYLD_LIBRARY_PATH=./libtexpdf/.libs ./sile tests/bug-75.sil`

No dice. It seems to work fine, but according to the history of this bug that combination should actually fail. What gives? Any idea what I'm missing?

**Comment by McSinyx** [comment]
On the other hand, I'm running into this trying to use [Noto Color Emoji](https://github.com/googlefonts/noto-emoji).  [Blobmoji](https://github.com/C1710/blobmoji) (a fork of Noto Emoji) also suffers the same fate.  For some reason, specifying just `Noto Emoji` works but it's the old blob glyphs.

I'm on SILE 0.12 on NixOS.
